docs/reference/docs/reindex.asciidoc

```diff
@@ -203,10 +205,10 @@ POST _reindex
 // CONSOLE
 // TEST[setup:twitter]
 
-If you want a particular set of documents from the twitter index you'll
-need to sort. Sorting makes the scroll less efficient but in some contexts
+If you want a particular set of documents from the `twitter` index you'll
+need to `sort`. Sorting makes the scroll less efficient but in some contexts
 it's worth it. If possible, prefer a more selective query to `size` and `sort`.
-This will copy 10000 documents from `twitter` into `new_twitter`:
+This will copy 10,000 documents from `twitter` into `new_twitter`:
```

Comment:
nit: I'd leave this as is, the comma is used in many countries (e.g. in Germany and most of the EU) as the decimal separator, so to me personally this reads more like 10 (with a weird extra zero). The existing version might not be as good to read but is less ambiguous I think.

Reply:
Agreed